docs/source/best-practices/checkpointing.rst

```diff
@@ -40,17 +40,17 @@ Implementing checkpointing can be acheived by:
  * The use of `Slurm Job Arrays <https://slurm.schedmd.com/job_array.html>`_. 
 
 .. note::
-   To overcome partition time limits, replace your single long job with multiple shorter jobs. Using job arrays, set each job to run one after the other. Each job will write a checkpoint file if checkpointing is implemented. The next job in line will be the latest checkpoint file to continue from the latest state of the calculation.
+   To overcome partition time limits, or to use the lowpriority partition effectively, replace your single long job with multiple shorter jobs. Using job arrays, set each job to run one after the other. Each job will write a checkpoint file if checkpointing is implemented. The next job in line will be the latest checkpoint file to continue from the latest state of the calculation.
```

Comment:
I am not sure this sentence reads correctly, should it be 'and to use the lowpriority' instead of 'or'? Might be worth rephrasing.

Reply:
thanks, changed that.